ion-transformation-storage-api/src/main/java/com/connexta/transformation/storage/api/IntelInfo.java

```diff
@@ -0,0 +1,54 @@
+/**
+ * Copyright (c) Connexta
+ *
+ * <p>This is free software: you can redistribute it and/or modify it under the terms of the GNU
+ * Lesser General Public License as published by the Free Software Foundation, either version 3 of
+ * the License, or any later version.
+ *
+ * <p>This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public
+ * License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package com.connexta.transformation.storage.api;
+
+import java.time.Instant;
+
+public interface IntelInfo {
```

Comment:
❗️ I am not sure we need the distinction between intel info and metadata info here. The intel info is what you return as part of the transform info